src/jit/unwind.h

```diff
@@ -385,6 +385,7 @@ class UnwindEpilogCodes : public UnwindBase, public UnwindCodesBase
         noway_assert(0 <= uecCodeSlot && uecCodeSlot < uecMemSize);
 
         uecMem[uecCodeSlot] = b;
+        if (firstCode) lastCode = b;
```

Comment:
Rather than creating a wierd interface for AppendByte, could you just move the lastCode maintenance to each of the AddCode overloads?  Also, I'd rename lastCode to firstByteOfLastCode or something like that.